libbeat/common/docker/watcher.go

```diff
@@ -276,7 +276,7 @@ func (w *watcher) watch() {
 			select {
 			case event := <-events:
 				w.log.Debugf("Got a new docker event: %v", event)
-				lastValidTimestamp = time.Unix(event.Time, event.TimeNano)
+				lastValidTimestamp = time.Unix(0, event.TimeNano)
```

Comment:
Btw, `timeNano` was introduced in docker API 1.21. This is quite old, but if we want to support these versions we should take this into account here.
```suggestion
                                if event.TimeNano > 0 {
                                    lastValidTimestamp = time.Unix(0, event.TimeNano)
                                } else {
                                    lastValidTimestamp = time.Unix(event.Time, 0)
                                }
```

Reply:
Nice! thank you for checking this!